src/main/java/se/nackademin/Inventory.java

```diff
@@ -0,0 +1,80 @@
+package se.nackademin;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Inventory {
+    private String vendor;
+    private boolean vendorExists;
```

Comment:
This variable was not obvious to me what it represents, some Javadoc would help.